rzup/install

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# From https://github.com/foundry-rs/foundry/blob/master/foundryup/install
+
+set -eo pipefail
+
+echo "Installing rzup..."
+
+BASE_DIR=${XDG_CONFIG_HOME:-$HOME}
+RISC0_DIR=${RISC0_DIR-"$BASE_DIR/.risc0"}
+RISC0_BIN_DIR="$RISC0_DIR/bin"
+
+# TODO: Change this path to main
+BIN_URL="https://raw.githubusercontent.com/risc0/risc0/hmrtn/rzup-installer/rzup/rzup"
```

Comment:
I suggest we host this on an S3 bucket to avoid github rate limiting.

Reply:
replaced with https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/rzup